unsize/src/lib.rs

```diff
@@ -60,7 +60,7 @@ mod impls {
 
     unsafe impl<Ptr, U : ?Sized, T> CoerciblePtr<U> for core::pin::Pin<Ptr>
     where
-        Ptr: CoerciblePtr<U> + core::ops::DerefMut<Target=T>,
+        Ptr: CoerciblePtr<U, Pointee=T> + core::ops::DerefMut<Target=T>,
```

Comment:
Oh nice I did wonder at some point about `Pin<impl Coercible…>` having two ways to mention the pointee 😄